src/AutoUI/index.tsx

```diff
@@ -74,6 +72,9 @@ import {
 } from '@balena/ui-shared-components';
 import type { FiltersView } from '../components/Filters';
 import { ajvFilter } from '../components/Filters/SchemaSieve';
+import { Format } from '../components/Widget/utils';
+import { Dictionary } from '../common';
```

Comment:
`type`?

Reply:
I'll try to update the linter as soon as possibile so that this will be automatic